Assets/Scripts/Menu.cs

```diff
@@ -5,9 +5,17 @@
 
 public class Menu : MonoBehaviour
 {
+    public void OnTutorialButton()
+    {
+        SceneManager.LoadScene("Tutorial-Power");
+    }
+    public void OnTutorialButton1()
+    {
+        SceneManager.LoadScene("Tutorial-emily");
```

Comment:
We should probably rename this tutorial scene or else it kind of implies its for testing purposes.